sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * soc-apci-intel-adl-match.c - tables and support for ADL ACPI enumeration.
+ *
+ * Copyright (c) 2020, Intel Corporation.
+ */
+
+#include <sound/soc-acpi.h>
+#include <sound/soc-acpi-intel-match.h>
+
+static const struct snd_soc_acpi_endpoint single_endpoint = {
+	.num = 0,
+	.aggregated = 0,
+	.group_position = 0,
+	.group_id = 0,
+};
+
+static const struct snd_soc_acpi_adr_device rt711_0_adr[] = {
+	{
+		.adr = 0x000021025D071100,
+		.num_endpoints = 1,
+		.endpoints = &single_endpoint,
+		.name_prefix = "rt711"
+	}
+};
+
+static const struct snd_soc_acpi_link_adr adl_rvp[] = {
```

Comment:
@kv2019i adr 0x000021025D071100 is used by AIOC codec card. It is not the RVP on-board codec. RVP on-board codec is using hda mode. I'm not sure which name is better, adl_rvp or adl_aioc?

Reply:
> @kv2019i adr 0x000021025D071100 is used by AIOC codec card. It is not the RVP on-board codec. RVP on-board codec is using hda mode. I'm not sure which name is better, adl_rvp or adl_aioc?

rt711 is a dual mode codec which can work as HDA or SDW mode. So I believe that the on-board codec can work as SDW mode with a proper reworking. And there are more than rt711 on the AIOC card. So I will vote "adl_rvp" as rt711, the on-board codec is the only codec in  adl_rvp[].

Reply:
ack @bardliao and @libinyang I'll adapt this to the onboard one and resubmit.

Reply:
@bardliao  @libinyang I now update the patch -- it should now match the onboard codec.